root/etc/e-smith/templates/etc/postfix/main.cf/40enableTLS

```diff
@@ -0,0 +1,27 @@
+
+#
+# TLS
+#
+
+# SMTPD
+smtpd_tls_security_level = may
+smtpd_tls_auth_only = no
+smtpd_tls_key_file = /etc/postfix/postfix.key
+smtpd_tls_cert_file = /etc/postfix/postfix.crt
+smtpd_tls_received_header = yes
+smtpd_tls_loglevel = 1
+smtpd_tls_session_cache_timeout = 3600s
+tls_random_source = dev:/dev/urandom
```

Comment:
Do not set options, if the value is already provided by default (i.e. see `postconf -d | grep tls_random_source`)